src/runtime/runtime.ts

```diff
@@ -113,8 +113,19 @@ const getSubcommand = (spec?: Fig.Spec): Fig.Subcommand | undefined => {
 const executeShellCommand = buildExecuteShellCommand(5000);
 
 const genSubcommand = async (command: string, parentCommand: Fig.Subcommand): Promise<Fig.Subcommand | undefined> => {
-  const subcommandIdx = parentCommand.subcommands?.findIndex((s) => s.name === command);
-  if (subcommandIdx == null) return;
+  if (!parentCommand.subcommands || parentCommand.subcommands.length === 0) return;
+  
+  const subcommandIdx = parentCommand.subcommands?.findIndex((s) => {
+    // subCommand.name can be a string or an array of strings
+    if (Array.isArray(s.name)) {
+      // if it's an array, we check if the command is included in the array
+      return s.name.includes(command);
+    } else {
+      return s.name === command;
+    }
+  });
```

Comment:
```suggestion
  const subcommandIdx = parentCommand.subcommands.findIndex((s) => Array.isArray(s.name) ? s.name.includes(command) : s.name === command);
```
More personal preference, but could we use a ternary here instead of an if / else? We also can remove the nullish access on subcommands since we've enforced it exists above.